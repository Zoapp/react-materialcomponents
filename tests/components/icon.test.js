import React from "react";
import renderer from "react-test-renderer";
import Icon from "@libs/components/icon";

describe("components/Icon", () => {
  it("renders correctly", () => {
    const component = renderer.create(<Icon name="foo" />);

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
