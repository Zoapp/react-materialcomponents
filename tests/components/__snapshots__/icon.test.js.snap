// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Icon renders correctly 1`] = `
<i
  className="material-icons"
>
  foo
</i>
`;
